package.json

```diff
@@ -5,7 +5,7 @@
   "scripts": {
     "dev": "nf start",
     "build": "vue-cli-service build",
-    "lint": "vue-cli-service lint --no-fix --max-warnings 0 src/ bin/ test/ *.js",
+    "lint": "eslint --cache --no-fix --max-warnings 0 --ext .js,.vue src/ bin/ test/ *.js",
```

Comment:
I see that Vite also specifies `.cjs,.mjs` for `--ext`. We don't have any of those files currently, but maybe we will in the future?

Reply:
I thought about adding these, but I think we can wait until we have examples of them in the codebase. We haven't needed them yet.